filestore/handlers.py

```diff
@@ -38,7 +38,11 @@ def __call__(self, point_number):
                                           file_number)
                 spe_obj = PrincetonSPEFile(fname)
                 self._f_cache[file_number] = spe_obj
-            out_stack.append(self._f_cache[file_number][frame_no])
+            spe = self._f_cache[file_number]
+            if len(spe) > 1:
```

Comment:
@stuwilkins I thought you said we were only going to get 1 frame per SPE file.  If there is more than one don't we need to do a bit more sophisticated counting?